discovery.md

```diff
@@ -0,0 +1,16 @@
+# Box-client discovery: two options
+
+## Option 1: --no-cloud
```

Comment:
People who don't want to leak their data to the cloud, and who don't want to rely on cloud services for the continued operation of their foxbox. In fact, if we would apply item 4 of the Mozilla manifesto, we should not even be calling this an 'option'.